### PR TITLE
upgrade duckbase for query support

### PR DESCRIPTION
**Pull request by WadePeterson**

### CODE REVIEW

File: src/app/containers/calendarContainer.js [hunk 1]
```diff
@@ -41,15 +41,14 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
...
-
 export default compose(
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
-  firebaseConnect(mapPropsToPaths),
+  firebaseConnect((props, query) => {
+    const startDateKey = DateUtils.firebaseYearMonthDateFormat(DateUtils.addMonths(props.refDate, -1));
+    const endDateKey = DateUtils.firebaseYearMonthDateFormat(DateUtils.addMonths(props.refDate, 1));
+    return [
+      `/recipes/${props.accountId}`,
+      query.name('scheduledRecipes').ref(`/scheduledRecipes/${props.accountId}`).orderByKey().startAt(startDateKey).endAt(endDateKey)
```
Comment (mfernandez-rally) [inline src/app/containers/calendarContainer.js]: Eventually we will need this to be broken out, probably, when we start allowing view by week or day, but for now it's fine!

File: src/app/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ store.dispatch(FirebaseActions.listenToUser()).then(() => {
   ReactDOM.render(
     <MuiThemeProvider>
       <Provider store={store}>
-        <FirebaseProvider firebaseApp={firebaseApp}>
+        <FirebaseProvider firebaseApp={firebaseApp} store={store}>
```
Comment (mfernandez-rally) [inline src/app/index.js]: Can it not get the store from the parent `Provider`? I'm not sure how wrapping parents work here

**Comment by mfernandez-rally** [comment]
⭕️ 🇲🇲 🐐 , 👀 @ 🚯 💤 (❓❔)

File: src/app/index.js (hunk 2 above)
Comment (WadePeterson) [inline src/app/index.js]: It could, but I wanted to remove duckbase's dependency on `react-redux`

**Pull request merged**